### PR TITLE
Ben Feole 

### PR DESCRIPTION
**Pull request by bfeole**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,74 +1,141 @@
...
-    <section class="content-destination">
-      <h2>Pick Your Destination</h2>
-      <p>Expedition excursion design darn excursion fun, clean simple organized WordPress Travel colorful webdesign. Traveler blogger website design expedition clean excursion traveling.</p>
-      <img src="img/destination.jpg" alt="Second slide">
-    </section>
-    <section class="content-pick">
+  <body>
+    <div class="container home">
+      <header>
+        <div class="logo-txt">
```
Comment (Fractured2K) [inline index.html:25]: You're overall HTML is amazing! Keep up the good work!

File: css/index.css [hunk 2]
```diff
@@ -1 +1,348 @@
...
+  width: 48%;
+  padding-left: 1%;
+}
+@media (max-width: 500px) {
+  .home .content-section .img-content {
+    width: 100%;
+    object-fit: cover;
+  }
+}
+.home .content-section .img-content img {
```
Comment (Fractured2K) [inline css/index.css:281]: When selecting raw tags like the `img` tag here for example I would personally use `>` to be as specific as possible to make sure I don't accidentally mess with other tags.

Comment (bfeole) [reply css/index.css:281]: ah gotcha, I had it nested in the less file so I thought it would be specific enough.

**Review by Fractured2K — APPROVED** [review]
Beautiful code as always Ben! Keep it up, can't want to see you work wonders in JavaScript!

File: css/index.css (hunk 2 above)
Comment (Fractured2K) [reply css/index.css:281]: That makes sense. I was reading it as it was `CSS` haha.

**Pull request merged**